recipes/protobuf/all/conanfile.py

```diff
@@ -94,13 +124,13 @@ def generate(self):
         tc.cache_variables["protobuf_BUILD_LIBPROTOC"] = self.settings.os != "tvOS"
         tc.cache_variables["protobuf_DISABLE_RTTI"] = not self.options.with_rtti
         if is_msvc(self) or self._is_clang_cl:
-            runtime = msvc_runtime_flag(self)
-            if not runtime:
-                runtime = self.settings.get_safe("compiler.runtime")
-            tc.cache_variables["protobuf_MSVC_STATIC_RUNTIME"] = "MT" in runtime
+            tc.cache_variables["protobuf_MSVC_STATIC_RUNTIME"] = is_msvc_static_runtime(self)
         if is_apple_os(self) and self.options.shared:
             # Workaround against SIP on macOS for consumers while invoking protoc when protobuf lib is shared
             tc.variables["CMAKE_INSTALL_RPATH"] = "@loader_path/../lib"
+        tc.variables["protobuf_ABSL_PROVIDER"] = "package"
+        if not valid_min_cppstd(self, self._min_cppstd):
+            tc.variables["CMAKE_CXX_STANDARD"] = self._min_cppstd
```

Comment:
Why need to enforce the C++ now? Is C++14 missing in Protobuf CMake?

Reply:
@toge Do you remember why this was needed?

Reply:
Conan v1 fails without it:
```
[  2%] Built target utf8_range
protobuf/4.25.3: 
In file included from /Users/jenkins/workspace/prod-v1/bsr/42822/aaffe/.conan/data/protobuf/4.25.3/_/_/build/73b28dd12161ea11293d28fd8ed2d7d3910330ef/src/third_party/utf8_range/utf8_validity.cc:18:
In file included from /Users/jenkins/workspace/prod-v1/bsr/42822/aaffe/.conan/data/protobuf/4.25.3/_/_/build/73b28dd12161ea11293d28fd8ed2d7d3910330ef/src/third_party/utf8_range/utf8_validity.h:10:
In file included from /Users/jenkins/workspace/prod-v1/bsr/42822/aaffe/.conan/data/abseil/20230802.1/_/_/package/f1a36a2aea2da1148eb4bddd758d8db183c6db83/include/absl/strings/string_view.h:39:
In file included from /Users/jenkins/workspace/prod-v1/bsr/42822/aaffe/.conan/data/abseil/20230802.1/_/_/package/f1a36a2aea2da1148eb4bddd758d8db183c6db83/include/absl/base/attributes.h:37:
In file included from /Users/jenkins/workspace/prod-v1/bsr/42822/aaffe/.conan/data/abseil/20230802.1/_/_/package/f1a36a2aea2da1148eb4bddd758d8db183c6db83/include/absl/base/config.h:86:
/Users/jenkins/workspace/prod-v1/bsr/42822/aaffe/.conan/data/abseil/20230802.1/_/_/package/f1a36a2aea2da1148eb4bddd758d8db183c6db83/include/absl/base/policy_checks.h:79:2: error: "C++ versions less than C++14 are not supported."
#error "C++ versions less than C++14 are not supported."
```

Reply:
Okay, they pass explicitly the standard and is also documented to use the cxx standard

- https://github.com/protocolbuffers/protobuf/blob/v4.25.3/.github/workflows/test_cpp.yml#L120
- https://github.com/protocolbuffers/protobuf/blob/v4.25.3/cmake/README.md#c-version